app/serializers/developer.js

```diff
@@ -0,0 +1,27 @@
+import RESTSerializer from 'ember-data/serializers/rest'
```

Comment:
It seems to me that if you created your backend properly, according to JSON-API standards, this file would not be necessary. Typically, you have to write serializers like this in Ember because we're dealing with a backend that is problematic, but since you've written the backend your own self, it's better for you to fix the backend than to write this code.